plugins/mcreator-localization/lang/texts.properties

```diff
@@ -2510,6 +2510,8 @@ elementgui.block.visual_rendering=Visual rendering settings
 elementgui.block.slipperiness=<html>Slipperiness:<br><small>Normal slipperiness is 0.6
 elementgui.block.speed_factor=<html>Speed factor:<br><small>Normal speed factor is 1.0
 elementgui.block.jump_factor=<html>Jump factor:<br><small>Normal jump factor is 1.0
+elementgui.common.bounciness=<html>Is bouncy?:<br><small>Must have collision/ be solid to be bouncy
```

Comment:
Generally we avoid second small line in procedure selectors as it makes them quite tall. I recommend moving info from the second line to the help tip, applies to both texts